servers/Azure.Mcp.Server/README.md

```diff
@@ -170,6 +218,8 @@ AZURE_CLIENT_SECRET={YOUR_AZURE_CLIENT_SECRET}
 ```
 
 </details>
+<!-- remove-section: end -->
```

Comment:
Are both of these needed? If so add a comment on each with something unique that matches the start-section

Can the add-section and remove-section comment tags support comments within them? It'll help people find matching pairs

Reply:
I have added comment tags to both start and remove